src/screens/delegation-screen/selected-baker-screen/baker-rewards-list/baker-reward-item/baker-reward-item-details/details/endorsements.tsx

```diff
@@ -2,6 +2,8 @@ import { BigNumber } from 'bignumber.js';
 import React, { FC } from 'react';
 import { Text, View } from 'react-native';
 
+import { useNetworkInfo } from 'src/hooks/use-network-info.hook';
+
 import { Divider } from '../../../../../../../components/divider/divider';
 import { Icon } from '../../../../../../../components/icon/icon';
 import { IconNameEnum } from '../../../../../../../components/icon/icon-name.enum';
```

Comment:
It would be better to write these and other similar imports from other files with `src/` as well